Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,34 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol.git", :tag => "v2.2.1"
```

Comment:
The cookbook needs updating, thanks. We have both to make sure the tag doesn't change.